src/anm/animation/tween.js

```diff
@@ -259,7 +260,7 @@ Tween.register(C.T_ROT_TO_PATH, {
             var path = this.$mpath;
             // when t equals exact 0, it is replaced with 0.001
             // or else returned angle would be 0
-            if (path) this.angle += path.tangentAt(t || 0.001);
+            if (path) this.angle += path.tangentAt(this.$mpath_t || 0.001);
```

Comment:
It needs to be something _other than 0_ in case if `mpath_t` is 0, so i.e. in very first frame or the animation, the shape will be properly oriented to path (there were several issues on when player shows first frame wrong in "thumbnail" (at time `0`) because shape is rotated to 0 degrees while it shouldn't).